governance/roles.md

```diff
@@ -145,8 +145,29 @@ and led by a Security Assessment Facilitator, who will:
   of significant process changes).
 * triage issues related to security assessments.
 
-
-### Triage Team
+### Meeting facilitator
+
+The group meetings are an important part of community building and the 
+facilitator ensures a welcoming and inclusive atmosphere. In keeping with these
+goals, the meeting facilitator has the following responsibilities:
+* prepares the meeting notes with template and agenda
+* coordinates with presenter, if applicable
+* ensures there are scribes
+* runs meeting check-in, including partner groups
+* leads the meeting through the agenda
+
+Prerequisites:
+* active member
+* history of regular attendance
+* participation as scribe
```

Comment:
I'm a little unsure if I parse this correctly... does this mean "Has participated as a scribe before" or "Has to participate as a scribe during the meeting that one is facilitating"?